### DISCUSSION TITLE
How to implement an animated splash screen

### DISCUSSION THREAD

**Discussion by CYS-xinjie** [appcelerator/titanium_mobile#13745] [Q&A] [answered]
Hi all, here I would like to ask whether the titanium sdk can support animated splash screen? I will be very grateful if I can receive your reply and help, below is the sample
https://dribbble.com/shots/4218698-Splash-screen

**Comment by m1ga** [comment, marked as answer]
You've contacted me on two different accounts so you should have an email already :wink: 

short: check Androids new splash screen animation xmls; use a first window to display the animation

**Reply by CYS-xinjie** [reply]
thanks a lot for your reply :)
